www/js/onboarding/onboardingHelper.ts

```diff
@@ -27,9 +27,10 @@ export const setRegisterUserDone = (b) => registerUserDone = b;
 
 export function getPendingOnboardingState(): Promise<OnboardingState> {
   return Promise.all([getConfig(), readConsented(), readIntroDone()]).then(([config, isConsented, isIntroDone]) => {
-    if (isIntroDone) return null; // onboarding is done; no pending state
-    let route: OnboardingRoute = OnboardingRoute.NONE;
-    if (!config) {
```

Comment:
just for my edification, we already had a route of `NONE`. 
It is apparently not used for anything else.
So couldn't you have just used it to distinguish between unknown and intro not done?

Reply:
Yes I considered that, but I reasoned that a state of `NONE` can be ambiguous to someone looking at the codebase with fresh eyes. (Is the state NONE because onboarding is over? Or because it hasn't started yet? Or we don't even know yet?)

But `DONE` is self-evident and unambiguous in its meaning. We also already use the word 'done' through this file.